docs/getting-started.md

```diff
@@ -34,15 +34,25 @@ brew install momento-cli
 
 **Linux**
 
-1. Download the latest linux tar.gz file from [https://github.com/momentohq/momento-cli/releases/download/v0.22.8/momento-cli-0.22.8.linux_x86_64.tar.gz](https://github.com/momentohq/momento-cli/releases/download/v0.22.8/momento-cli-0.22.8.linux_x86_64.tar.gz)
-2. Extract the `momento-cli-X.X.X.linux_x86_64.tar.gz` file
-3. Move `./momento` to your execution path.
+Visit the web page for the latest [github release](https://github.com/momentohq/momento-cli/releases).
+There, you will find `.deb` and `.rpm` files for both x86_64 and aarch64.
+
+`.deb` files have been tested on modern versions of Ubuntu and Debian.
+`.rpm` files have been tested on modern versions of RHEL, Amazon Linux 2, Rocky Linux, and CentOS.
+
+If you have problems with any of these packages on your favorite platform, please [file an issue](https://github.com/momentohq/momento-cli/issues) and let us know!
+
+We also provide tarballs for both x86_64 and aarch64; these contain the `momento` binary,
+which you may add anywhere you like in your execution path.
 
 **Windows**
+Visit the web page for the latest [github release](https://github.com/momentohq/mom
```

Comment:
Missing newline after **Windows** puts this all on the same line. Use `### Windows` heading instead?

Reply:
nice catch thanks

Reply:
but I don't want to switch this file from stars to pounds because it goes through some doc generator tool that i'm not familiar with, so just going to add the newline.